### PR TITLE
docs(release): update changelog for 11-2023 release

### PR DESCRIPTION
**Pull request by alevenberg**
This is what the script produced:

```
❯ release/changes.sh


### [BigQuery](/google/cloud/bigquery/README.md)

- fix(bigquery): propagate bad status ([#12973](https://github.com/googleapis/google-cloud-cpp/pull/12973))
- fix(generator): do not override default sleeper in streaming-read RPCs ([#12920](https://github.com/googleapis/google-cloud-cpp/pull/12920))

### [OpenTelemetry](/google/cloud/opentelemetry/README.md)

- doc(otel): add packaging instructions ([#13005](https://github.com/googleapis/google-cloud-cpp/pull/13005))

### [Pub/Sub](/google/cloud/pubsub/README.md)

- Revert "impl(pubsub): use tracing message batch" ([#12998](https://github.com/googleapis/google-cloud-cpp/pull/12998))
- fix(pubsub): create root async batch sink spans ([#12975](https://github.com/googleapis/google-cloud-cpp/pull/12975))
- fix(pubsub): add missing default capture ([#12984](https://github.com/googleapis/google-cloud-cpp/pull/12984))

### [Spanner](/google/cloud/spanner/README.md)

- feat(spanner): single-RPC, batched commit of mutation groups ([#12930](https://github.com/googleapis/google-cloud-cpp/pull/12930))

### [Storage](/google/cloud/storage/README.md)

- fix(common): missed using `CARootsFilePathOption` ([#12997](https://github.com/googleapis/google-cloud-cpp/pull/12997))
- fix(storage): fix when async client spans are ended ([#12921](https://github.com/googleapis/google-cloud-cpp/pull/12921))
- fix(storage): remove unused dependencies ([#12890](https://github.com/googleapis/google-cloud-cpp/pull/12890))

### [Common Libraries](/google/cloud/README.md)

- feat(redis): add cluster ([#13004](https://github.com/googleapis/google-cloud-cpp/pull/13004))
- fix(common): missed using `CARootsFilePathOption` ([#12997](https://github.com/googleapis/google-cloud-cpp/pull/12997))
- fix(otel): support abseil <= 20210324 ([#12993](https://github.com/googleapis/google-cloud-cpp/pull/12993))
- fix(otel): restore compatibility with otel 1.9.1 ([#12992](https://github.com/googleapis/google-cloud-cpp/pull/12992))
- feat(common): mock for `AsyncStreamingReadWriteRpc` ([#12950](https://github.com/googleapis/google-cloud-cpp/pull/12950))
- fix(storage): fix when async client spans are ended ([#12921](https://github.com/googleapis/google-cloud-cpp/pull/12921))
- fix(generator): do not override default sleeper in streaming-read RPCs ([#12920](https://github.com/googleapis/google-cloud-cpp/pull/12920))
- fix(otel): end async client spans properly ([#12919](https://github.com/googleapis/google-cloud-cpp/pull/12919))
- feat(securesourcemanager): generate library ([#12895](https://github.com/googleapis/google-cloud-cpp/pull/12895))
```

I trimmed what I thought was not necessary

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13012)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,23 @@ the APIs in these libraries are stable, and are ready for production use.
...
+### [Spanner](/google/cloud/spanner/README.md)
+
+- feat(spanner): single-RPC, batched commit of mutation groups
+  ([#12930](https://github.com/googleapis/google-cloud-cpp/pull/12930))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- feat(common): mock for `AsyncStreamingReadWriteRpc`
+  ([#12950](https://github.com/googleapis/google-cloud-cpp/pull/12950))
+
```
Comment (dbolduc) [inline CHANGELOG.md:41]: Add:
```
- fix(common): missed using `CARootsFilePathOption` ([#12997](https://github.com/googleapis/google-cloud-cpp/pull/12997))
- fix(generator): do not override default sleeper in streaming-read RPCs ([#12920](https://github.com/googleapis/google-cloud-cpp/pull/12920))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,23 @@ the APIs in these libraries are stable, and are ready for production use.
 
 New `CloudRedisClusterClient`.
 
+### [OpenTelemetry](/google/cloud/opentelemetry/README.md)
+
```
Comment (dbolduc) [inline CHANGELOG.md:26]: Add:
```
- fix(otel): support abseil <= 20210324 ([#12993](https://github.com/googleapis/google-cloud-cpp/pull/12993))
```

**Review by dbolduc — COMMENTED** [review]
Good idea to show the output of `release/changes.sh`

File: CHANGELOG.md (hunk 1 above)
Comment (alevenberg) [reply CHANGELOG.md:44]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (alevenberg) [reply CHANGELOG.md:26]: Done

**Review by dbolduc — APPROVED** [review]

**Review by devbww — APPROVED** [review]

**Comment by alevenberg** [comment]
Double approvals. Nice 🆒

**Pull request merged**